extensions/r2-uploader/package.json

```diff
@@ -0,0 +1,130 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "r2-uploader",
+  "title": "Cloudflare R2 File Uploader",
+  "description": "Upload any files to Cloudflare R2 with optional AVIF conversion for images. After upload, the URL or Markdown link will be copied to clipboard.",
+  "icon": "extension-icon.png",
+  "author": "Silas",
+  "categories": [
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "type": "module",
```

Comment:
Inconsistent configuration. You set the `type: module`, but the `tsconfig.json` is still using `commonjs`.